src/main/kotlin/no/nav/familie/tilbake/iverksettvedtak/IverksettelseService.kt

```diff
@@ -65,19 +69,24 @@ class IverksettelseService(
                 info("Sender tilbakekrevingsvedtak til økonomi for behandlingId={} request={}", behandlingId.toString(), requestXml)
             }
 
-            // Send request til økonomi
-            val kvittering = objectMapper.writeValueAsString(oppdragClient.iverksettVedtak(behandlingId, request, logContext).mmel)
-            lagreIverksettelsesvedtakRequest(behandlingId, requestXml, kvittering)
+            val mmel: MmelDto = oppdragClient.iverksettVedtak(behandlingId, request, logContext).mmel
```

Comment:
Jeg tror kanskje det enkleste blir å beholde dette som det er. Da slipper vi en overlapp periode hvor vi har gammel og ny iverksettelse. Om vi skal bytte ut dette on the go må vi rulle det ut en dag saksbehandlere ikke jobber, og vi kan ikke enkelt rulle tilbake